### PR TITLE
Created the UK-Translation

### PR DESCRIPTION
**Pull request by RedstoneDubstep**
I tried to fix the annoying grey/gray issue of the reinforced grey/gray blocks.
If there is any problem, please contact me.
I hope that I finally got the pull request working now!

### CODE REVIEW

File: src/main/resources/assets/securitycraft/lang/en_uk.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+        "block.securitycraft.reinforced_gray_stained_glass": "Reinforced Grey Stained Glass",
+        "block.securitycraft.reinforced_light_gray_stained_glass": "Reinforced Light Grey Stained Glass",
+        "block.securitycraft.reinforced_gray_stained_glass_pane": "Reinforced Grey Stained Glass Pane",
+        "block.securitycraft.reinforced_light_gray_stained_glass_pane": "Reinforced Light Gray Stained Glass Pane",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_uk.json]: `Gray` -> `Grey`

Comment (RedstoneDubstep) [reply src/main/resources/assets/securitycraft/lang/en_uk.json]: ok, I'm gonna change that, my bad

Comment (RedstoneDubstep) [reply src/main/resources/assets/securitycraft/lang/en_uk.json]: but I've searched the wiki list and I've not found a difference in the words which is actually in this language file

Comment (RedstoneDubstep) [reply src/main/resources/assets/securitycraft/lang/en_uk.json]: do you know a word I might have forgotten?

Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/en_uk.json]: No, I don't. I just wanted you to check whether there are any other things to change for British English.

**Pull request merged**